src/Microsoft.PowerShell.Commands.Management/commands/management/GetClipboardCommand.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System;
+﻿#if !CORECLR
```

Comment:
`GetClipboardCommand.cs` and `SetClipboardCommand.cs` are currently not included in the build (see [here](https://github.com/PowerShell/PowerShell/blob/master/src/Microsoft.PowerShell.Commands.Management/Microsoft.PowerShell.Commands.Management.csproj#L32)). We use those excluded compile entries in `csproj` files as a living document to track what built-in cmdlets are not in powershell core yet. So the `if/def CORECLR` is not necessary in those files.